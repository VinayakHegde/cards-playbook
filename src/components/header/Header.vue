<script src="./index.js"></script>
<!-- Add "scoped" attribute to limit CSS to this component only -->
<style scoped lang="scss" src="./style.scss"></style>
<template>
  <header>{{ text }}</header>
</template>
