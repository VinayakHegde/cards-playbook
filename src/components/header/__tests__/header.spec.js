import { mount } from "@vue/test-utils";
import Header from "../Header.vue";
describe("Header.vue", () => {
  const wrapper = mount(Header);
  it("match the snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
