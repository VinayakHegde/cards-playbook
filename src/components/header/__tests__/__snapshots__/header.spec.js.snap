// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header.vue match the snapshot 1`] = `<header></header>`;
